src/lib/viewers/controls/annotations/AnnotationsTargetedTooltip.tsx

```diff
@@ -16,9 +16,9 @@ function AnnotationsTargetedTooltip({ children, isEnabled = false }: Props): JSX
     return (
         <TargetedClickThroughTooltip
             className="bp-AnnotationsTooltip"
+            closeOnClickOutside
             shouldTarget={isEnabled}
             showCloseButton
-            tetherElementClassName="bp-AnnotationsTooltipTether"
```

Comment:
we no longer need this as the tooltip will close if the preview share modal is clicked